src/main/java/com/marianbastiurea/lifeofbees/Hive.java

```diff
@@ -1,24 +1,42 @@
 package com.marianbastiurea.lifeofbees;
 
-public  class Hive {
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+
+public class Hive {
     private int id;
     private int numberOfHoneyFrame;
     private int numberOfEggsFrame;
+    private int ageOfQueen;
+    private EggsFrame eggsFrame;
+    private HoneyFrame honeyFrame;
+    private int numberOfBees;
+    private double honeyQuantity;
     private Queen queen;
+    private Bees bees;
+    private List<EggsBatch> eggsBatches;
```

Comment:
Instantiate `eggsBatches` in the constructor.

Reply:
done